server/dash/dashboards/ncSumComp.py

```diff
@@ -0,0 +1,407 @@
+import datetime
+import dash_daq as daq
+import numpy as np
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
+import requests as re
+
+from config import API_HOST
+from design import DISCRETE_COLORS, LABELS, apply_figure_style
+from dash import dcc, html, callback
+from dash.dependencies import Input, Output
+from dash.exceptions import PreventUpdate
+
+# Setting 1 week worth of data
+# TODO: reset back to 1 week worth of data
+start_date = datetime.date.today() - datetime.timedelta(days=300)
+end_date = datetime.date.today() - datetime.timedelta(days=200)
+
+# Loading the dataframe with the 311 Data API
+DATE_RANGE_REQ_DATA_API_PATH = f"/requests/updated?start_date={start_date}&end_date={end_date}"
+print(" * Downloading data for dataframe from API path: " + DATE_RANGE_REQ_DATA_API_PATH)
+results = re.get(API_HOST + DATE_RANGE_REQ_DATA_API_PATH)
+data_json = results.json()
+data_2020 = pd.json_normalize(data_json)
+print(" * Loading complete dataframe from API path: " + DATE_RANGE_REQ_DATA_API_PATH)
+
+# LAYOUT VARIABLES
+BORDER_STYLE = {"border": "0.5px black solid"}
+CENTER_ALIGN_STYLE = {"text-align": "center"}
+DIVIDER_STYLE = {"height": "1vh"}
+EQUAL_SPACE_STYLE = {"display": "flex", "justify-content": "space-between"}
+INLINE_STYLE = {"display": "inline-block"}
+CHART_OUTLINE_STYLE = INLINE_STYLE.update(BORDER_STYLE)
```

Comment:
Is this working as intended? `update` does not have a return value. https://www.geeksforgeeks.org/python-dictionary-update-method/

Reply:
hmm it seems to work on my end. Let me get back to this

Reply:
You're right. This doesn't return. Added merge_dict function to achieve original goal.